components/team.js

```diff
@@ -4,18 +4,21 @@ import TeamMember from './teamMember'
 const teamMembers = [
   {
     name: 'Joel Wasserman',
+    title: 'Outreach Director',
```

Comment:
what are these, why do we need titles

Reply:
They were in the design

Reply:
can we remove them, i think they were just a placeholder

Reply:
removed